ceno_zkvm/src/scheme/cpu/mod.rs

```diff
@@ -43,18 +44,11 @@ pub struct CpuTowerProver;
 
 impl CpuTowerProver {
     pub fn create_proof<'a, E: ExtensionField, PCS: PolynomialCommitmentScheme<E>>(
-        mut out_evals: Vec<Vec<Vec<E>>>,
```

Comment:
This out_evals seems introduce in target branch recently. This design is a bit weird because usually in prover we dont need to pass evaluation. Thus this api design is weird and we can left it handle externally, which is what master branch.